feature-libs/organization/administration/core/http-interceptors/conflict/conflict.handler.ts

```diff
@@ -0,0 +1,117 @@
+import { Injectable } from '@angular/core';
+import { HttpErrorResponse, HttpRequest } from '@angular/common/http';
+import {
+  ErrorModel,
+  GlobalMessageType,
+  HttpErrorHandler,
+  HttpResponseStatus,
+  Priority,
+} from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OrganizationConflictHandler extends HttpErrorHandler {
+  responseStatus = HttpResponseStatus.CONFLICT;
+
+  hasMatch(errorResponse: HttpErrorResponse): boolean {
+    return (
+      super.hasMatch(errorResponse) && this.getErrors(errorResponse).length > 0
+    );
+  }
+
+  handleError(request: HttpRequest<any>, response: HttpErrorResponse) {
+    if (!this.handleDuplicated(request, response)) {
```

Comment:
I think we should cover this case in `hasMatch`, if we are not able to handle the error, then `hasMatch` should return false, and leave fallback to the global mechanism.